lib/configs/documentation.ts

```diff
@@ -39,6 +39,11 @@ export function documentation(options: ConfigOptions): Linter.Config[] {
 				...GLOB_FILES_JAVASCRIPT,
 				...(options.vueIsTypescript ? [] : GLOB_FILES_VUE),
 			],
+			settings: {
+				jsdoc: {
+					mode: 'jsdoc',
+				},
+			},
```

Comment:
Does it still allow using TypeScript for typing and other features from TS in jsdoc?

Reply:
Yes added a test for it